corehq/apps/users/forms.py

```diff
@@ -23,12 +23,13 @@
 from corehq.apps.domain.models import Domain
 from corehq.apps.locations.models import SQLLocation
 from corehq.apps.locations.permissions import user_can_access_location_id
+from corehq.apps.settings.forms import EncodedPasswordChangeForm
```

Comment:
I think you want to change this to
```python
from custom.nic_compliance.forms import EncodedPasswordChangeForm
```
now, even though this still technically works (since `corehq.apps.settings.forms` also imports this). I'll also comment on the other places that I'd make the same change.